.github/workflows/e2e-windows.yml

```diff
@@ -0,0 +1,93 @@
+name: e2e test windows
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * 6" # At 03:00 on Saturday.
+
+jobs:
+  build-cli:
+    name: Build Windows CLI
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          ref: ${{ !github.event.pull_request.head.repo.fork && github.head_ref || '' }}
+
+      - name: Build CLI
+        uses: ./.github/actions/build_cli
+        with:
+          targetOS: "windows"
+          targetArch: "amd64"
+          enterpriseCLI: true
+          outputPath: "build/constellation.exe"
+
+      - name: Upload CLI artifact
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          path: "build/constellation.exe"
+          name: "constellation.exe"
```

Comment:
This should upload container images like the other e2e tests.

Reply:
Then I should also edit the config file afterwards to use the appropriate images I assume? Or what's the purpose of uploading them?

Reply:
Never mind. You are correct in the sense that when using nightlies / releases, this is not necessary.